### ISSUE TITLE
Yi Dome camera 1080p

### ISSUE THREAD

**Issue by Dani98b** [TheCrypt0/yi-hack-v4#225]
good evening everyone, I would like to know how I can, access the web interface on my Yi dome camera1080p I wrote the firmware put it inside the sd card but it doesn't blink me in yellow and then it returns normal to me because I don't know how to do any of you know how can I do I would like to access on rtsp, and many, others. Contents who can help me please?

**Comment by subasically** [comment]
I am having the same issue...

**Comment by subasically** [comment]
> I am having the same issue...

So I got mine working. I copied the files to the microsd card and the reset the camera. I then went through the normal setup by hooking the camera up to my wifi and then once that was done I was apple to access the camera but navigating to the cameras ip address.

**Comment by Dani98b** [comment]
Yes this Problem is too strog im a yi 1080p WiFi ot rtcp im to workig for my camera whts the Problem?
Von: Alen Subasic<mailto:notifications@github.com>
Gesendet: Montag, 11. Mai 2020 18:12
An: TheCrypt0/yi-hack-v4<mailto:yi-hack-v4@noreply.github.com>
Cc: Dani98b<mailto:daniele_vitali@hotmail.ch>; Author<mailto:author@noreply.github.com>
Betreff: Re: [TheCrypt0/yi-hack-v4] Yi Dome camera 1080p (#225)


I am having the same issue...

So I got mine working. I copied the files to the microsd card and the reset the camera. I then went through the normal setup by hooking the camera up to my wifi and then once that was done I was apple to access the camera but navigating to the cameras ip address.
